src/Paket.Core/BindingRedirects.fs

```diff
@@ -50,7 +50,14 @@ let internal setRedirect (doc:XDocument) bindingRedirect =
 
 /// Applies a set of binding redirects to a single configuration file.
 let private applyBindingRedirects bindingRedirects (configFilePath:string) =
-    let config = XDocument.Load(configFilePath, LoadOptions.PreserveWhitespace)
+    let config = 
+        try 
+            XDocument.Load(configFilePath, LoadOptions.PreserveWhitespace)
+        with
+        | :? System.Xml.XmlException as ex ->
```

Comment:
As usual: Paket just terminates. :-)
But this won't fix it, exception is re-thrown.

I think this is good info for non-verbose also: Code won't do anything in happy case scenario.

----- Alkuperäinen viesti -----
Lähettäjä: "Steffen Forkmann" <notifications@github.com>
Lähetetty: ‎27.‎3.‎2015 19:37
Vastaanottaja: "fsprojects/Paket" <Paket@noreply.github.com>
Kopio: "Tuomas Hietanen" <tuomas.hietanen@iki.fi>
Aihe: Re: [Paket] If Paket crashes to some invalid user file: let's displaythe file to user so he/she can fix it. (#730)

In src/Paket.Core/BindingRedirects.fs:
> @@ -50,7 +50,14 @@ let internal setRedirect (doc:XDocument) bindingRedirect =
>  
>  /// Applies a set of binding redirects to a single configuration file.
>  let private applyBindingRedirects bindingRedirects (configFilePath:string) =
> -    let config = XDocument.Load(configFilePath, LoadOptions.PreserveWhitespace)
> +    let config = 
> +        try 
> +            XDocument.Load(configFilePath, LoadOptions.PreserveWhitespace)
> +        with
> +        | :? System.Xml.XmlException as ex ->
What happens with other exceptions?
—
Reply to this email directly or view it on GitHub.